lib/effects/move_effect.dart

```diff
@@ -39,34 +37,25 @@ class MoveEffect extends PositionComponentEffect {
     _xOriginal = component.x;
     _yOriginal = component.y;
 
-    _xDistance = _size(destination.x, component.x);
-    _yDistance = _size(destination.y, component.y);
+    _xDistance = _distance(destination.x, component.x);
+    _yDistance = _distance(destination.y, component.y);
 
     _xDirection = _direction(destination.x, component.x);
     _yDirection = _direction(destination.y, component.y);
 
-    _travelTime = max(
-      _xDistance / speed,
-      _yDistance / speed,
-    );
+    final totalDistance = sqrt(pow(_xDistance, 2) + pow(_yDistance, 2));
+    travelTime = totalDistance / speed;
```

Comment:
Interesting, care to explain the rationale on this new calculation? I am curious haha

Reply:
The other one wasn't exact since it only took the longest of the sides instead of the hypotenuse, this one uses the pythagorean theorem to get the length of the hypotenuse.

Reply:
Cool, thanks